### DISCUSSION TITLE
Data Persistance in HF Spaces

### DISCUSSION THREAD

**Discussion by williamzebrowskI** [open-webui/open-webui#4447] [General]
I'm curious if data persistence is possible in huggingface spaces.  

I do have a data persist subscription in a Space that allows persistence across restarts however even with that,  my open web ui data does not persist. 

Appreciate any help.

Thanks.

**Comment by justinh-rahb** [comment]
Try setting environment variable `DATA_DIR=/data`

**Reply by williamzebrowskI** [reply]
That worked.  Thank you @justinh-rahb !
